app/models/pipeline_run_stage.rb

```diff
@@ -93,6 +106,7 @@ def update_job_status
       if job_hash['container'] && job_hash['container']['logStreamName']
         self.job_log_id = job_hash['container']['logStreamName']
       end
+      run_job if instance_terminated?(job_hash) # retry if necessary
```

Comment:
if instance_terminated?(job_hash)
run_job
return
end 

so you don't save twice

Reply:
Ok! Thanks